### DISCUSSION TITLE
Spell check in cypress

### DISCUSSION THREAD

**Discussion by KarthikModelon** [cypress-io/cypress#23362] [Questions and Help] [answered]
Can we check spellings in cypress?

**Comment by emilyrohrbough** [comment, marked as answer]
@KarthikModelon I recommended installing a spell checker extension in your code editor. I use VSCode and have [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) installed. It works great.
